src/css/Chart.css

```diff
@@ -0,0 +1,126 @@
+* {
```

Comment:
- [ ] Good job so far with your project it looks great and works as expected :clap:  But we have an issue with your design :cry: As you check on the project requirement, we expect you to work on mobile design and we can see some differences between your design and the original design. Please make sure to follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), especially the layout: composition and space between elements as requested for this project 👍 It is important to try to make your design similar to the original design for both the home and details page as a developer it is important to pay attention to details :+1: 

#### Figma design
![metrics_app](https://user-images.githubusercontent.com/6615394/195218382-33f7f77a-f88f-41ac-a8ce-795156a3a495.jpg)